backend/server.go

```diff
@@ -0,0 +1,183 @@
+package backend
+
+import (
+	rice "github.com/GeertJohan/go.rice"
+	"github.com/gomodule/redigo/redis"
+	"html/template"
+	"log"
+	"net/http"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+type Data struct {
+	People []Person
+	Awards []string
+}
+
+type Person struct {
+	Name string `redis:"name"`
+	Score float64 `redis:"score"`
+}
+
+type App struct {
```

Comment:
nit: Here we have a public struct with private members-- does the struct need to be reused in another package?